package-lock.json

```diff
@@ -1,8 +1,8921 @@
 {
   "name": "yokoso",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
This package-lock diff :eyes: Did you happen to update npm recently?

Apparently v2 for lockfile version should be backwards compatible so I think it should be ok for anyone using an older version :crossed_fingers: . We should keep this in mind if anyone starts getting package/install issues.

Reply:
omg such a good catch! We'll keep a note of that, I think it is backwards compatible as you said so fingers crossed no issues occur. I'll be more careful next time